config/initializers/zhe_badges.rb

```diff
@@ -0,0 +1,3 @@
+Dir[Rails.root + './app/badges/*.rb'].each do |file|
+  require file
+end
```

Comment:
The filename on this is interesting, where did `zhe` come from, is that to keep this loading last?